packages/desktop-client/src/components/reports/SaveReport.tsx

```diff
@@ -145,15 +160,14 @@ export function SaveReport({
             flexShrink: 0,
           }}
         >
-          {!report.id ? 'Unsaved report' : report.name}&nbsp;
+          {report.id === '' ? 'Unsaved report' : report.name}&nbsp;
         </Text>
         {savedStatus === 'modified' && <Text>(modified)&nbsp;</Text>}
         <SvgExpandArrow width={8} height={8} style={{ marginRight: 5 }} />
       </Button>
       {menuOpen && (
         <SaveReportMenu
```

Comment:
idea: would be nice to disable the choose reports button if we don't have any saved reports.